Center window on current screen

Norodix opened the pull request:
Changing the window size moves the window to the left monitor on multi-monitor setups.
This change keeps the window on the current monitor after centering.

Norodix commented:
I noticed there is even a relevant issue after fixing it for myself.

https://github.com/PunchablePlushie/godot-game-settings/issues/24

PunchablePlushie reviewed (changes requested):
Apologies for the late review. Please make sure everything is statically typed. So:
```
var screen: int = ...
var origin: Vector2i = ...
```
Other than that, everything's fine. Thanks for the fix!

Norodix commented:
Added type hints.
Also, turns out I introduced a small bug, but that is fixed now. At one place I used the window_get function with the screen id which is obviously incorrect.
